Library/Homebrew/diagnostic.rb

```diff
@@ -477,7 +477,7 @@ def check_user_curlrc
           If you have trouble downloading packages with Homebrew, then maybe this
           is the problem? If the following command doesn't work, then try removing
           your curlrc:
-            curl https://github.com
+            curl #{Formatter.url("https://github.com")}
```

Comment:
I would personally not bother highlighting this URL, as it's a command line argument to be copy-pasted together with the `curl` rather than a URL we expect users to check out in isolation (most, if not all, other instances where `Formatter.url` is used). But I don't feel overly strongly about this.